src/libs/actions/Report.ts

```diff
@@ -1357,7 +1357,7 @@ function readNewestAction(reportID: string, shouldResetUnreadMarker = false) {
  * Sets the last read time on a report
  */
 function markCommentAsUnread(reportID: string, reportActionCreated: string) {
-    if (reportID === '-1') {
+    if (reportID === '') {
```

Comment:
Please, follow rules described in the guide: https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#default-value-for-inexistent-ids

ID should not be and empty string